_posts/release/2022-12-01-accumulo-2.1.1.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Apache Accumulo 2.1.1
+sortableversion: '02.01.01'
+LTM: true
+draft: true
+---
+
+** DRAFT RELEASE NOTES **
+
```

Comment:
You don't need to include this. The `draft: true` front matter does this automatically.

```suggestion

```